src/vellum/workflows/nodes/bases/base_subworkflow_node/node.py

```diff
@@ -1,10 +0,0 @@
-from typing import ClassVar, Generic
-
-from vellum.workflows.nodes.bases import BaseNode
-from vellum.workflows.types.core import EntityInputsInterface
-from vellum.workflows.types.generics import StateType
-
-
-class BaseSubworkflowNode(BaseNode[StateType], Generic[StateType]):
-    # Inputs that are passed to the Subworkflow
-    subworkflow_inputs: ClassVar[EntityInputsInterface] = {}
```

Comment:
The only thing this class reused between workflow nodes was subworkflow_inputs. With this new feature, the type now differs between the two, meaning there's no reason now to have a base class between the two nodes